UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -192,22 +230,30 @@
           wrapper = $('<div />').addClass('wrapper edit-mode');
         }
         streetViewHandler = getStreetView();
+
+        var components;
+        if (busStopTypeSelected == 7 && (applicationModel.getSelectedTool() == 'AddPointAsset') && selectedMassTransitStopModel.getCurrentAsset().id === undefined)  {
```

Comment:
Upgrade to this:
```javascript
if (busStopTypeSelected == 7 && (applicationModel.getSelectedTool() == 'AddPointAsset') && _.isUndefined(selectedMassTransitStopModel.getCurrentAsset().id))  {

Reply:
@marquesfm Check modification on this code